prod-docker-compose.yml

```diff
@@ -7,6 +7,17 @@ volumes:
   app_data: {}
 
 services:
+  nginx:
+    image: nginx:1.19-alpine
+    restart: unless-stopped
+    network_mode: host
+    volumes:
+      - ./home/ubuntu/secrets/nginx_certs:/data/nginx/certs
```

Comment:
Either use relative paths or make sure that you can access the secrets from one level down e.g. `../secrets/..`

Reply:
And I guess that you hand maded those nginx certs. Can you create a bash script that will do all the steps that you did by hand? Could be useful to always do it with a deploy as we can end up with a fresh instance of VM and we have to remember to do that by hand again.

Reply:
I have already made a scripts that converts all these certificates